PyDSS/utils/utils.py

```diff
@@ -20,6 +20,12 @@
 logger = logging.getLogger(__name__)
 
 
+def check_redirect(file_name):
+    result = dss.run_command("Redirect {}".format(file_name))
```

Comment:
This isn't a request for a change, but I do suggest that you try to be consistent with your style.  f-strings vs `"{}".format()`, single quotes vs double quotes.  :)